mcproto/types.go

```diff
@@ -79,6 +79,8 @@ const (
 	PacketIdHandshake            = 0x00
 	PacketIdLogin                = 0x00 // during StateLogin
 	PacketIdLegacyServerListPing = 0xFE
+	PacketIdStatusRequest        = 0x00
+	PacketIdStatusPing           = 0x01
```

Comment:
Please call this `PacketIdPingRequest` to match the docs naming convention:

https://minecraft.wiki/w/Java_Edition_protocol/Packets#Ping_Request_(status)

I wasn't able to find a "status ping" when searching the page as is.